core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -24,6 +24,28 @@ oppia.factory('UrlInterpolationService', ['alertsService',
     return (typeof value === 'string') || (value instanceof String);
   };
 
+  var validateResourcePath = function(resourcePath) {
+    if (!resourcePath) {
+      alertsService.fatalWarning(
+        'Empty path passed in method.');
+      return null;
```

Comment:
Fair enough. I saw it being used elsewhere and thought that it may have some significance.